From c14cd3cdb7ba1317e2278015175f15ac43f32408 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Fri, 17 Jul 2020 15:05:03 +0200
Subject: [PATCH 2/2] zerotier: disable ASLR PIE

I tested it on Turris Omnia, where zerotier segfaults on start.
When I disabled ASLR PIE, it works. This is a workaround.
We should invest a little bit more time to see why it does not work with
ASLR PIE.

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 net/zerotier/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/zerotier/Makefile b/net/zerotier/Makefile
index 4413da2df..369482685 100644
--- a/net/zerotier/Makefile
+++ b/net/zerotier/Makefile
@@ -17,6 +17,7 @@ PKG_BUILD_DIR:=$(BUILD_DIR)/ZeroTierOne-$(PKG_VERSION)
 PKG_MAINTAINER:=Moritz Warning <moritzwarning@web.de>
 PKG_LICENSE:=BSL 1.1
 PKG_LICENSE_FILES:=LICENSE.txt
+PKG_ASLR_PIE:=0
 
 PKG_BUILD_PARALLEL:=1
 include $(INCLUDE_DIR)/package.mk
-- 
2.27.0

